ts/queue.ts

```diff
@@ -48,12 +48,12 @@ export default class Queue {
       expires,
       deadLetterExchange,
       deadLetterRoutingKey,
-      maxLength
+      maxLength,
+      arguments: { ...this.options.arguments }
```

Comment:
```suggestion
      arguments
```

If you destruture arguments in L42 you use it like this here